terraform/files/bin/create_cluster.sh

```diff
@@ -80,9 +63,6 @@ export OS_CLOUD=$(yq eval '.OPENSTACK_CLOUD' $CCCFG)
 # Ensure image is there
 wait_capi_image.sh "$1" || exit 1
 
-# Switch to capi mgmt cluster
-export KUBECONFIG=$HOME/.kube/config
-~/bin/mng_cluster_ns.inc
```

Comment:
This was historically placed here to reset to a known state after calling the "external_scripts".

Reply:
Understood. As the `kubeconfig` file is duplicated I guess this is no longer needed.